### PR TITLE
List external geocoding services and rate limits

### PR DESCRIPTION
**Pull request by malandrina**

### CODE REVIEW

File: book/strategies/external_services.md [hunk 1]
```diff
@@ -1,6 +1,42 @@
 ## External Services
 
-* [Google](https://developers.google.com/maps/documentation/geocoding/)
-* [Yandex](http://api.yandex.com/maps/doc/jsapi/2.x/ref/reference/geocode.xml)
+### Choosing a service
+* Rate limits
+* Security
```
Comment (malandrina) [inline book/strategies/external_services.md]: Should we cover this? I'm not sure.

File: book/strategies/external_services.md [hunk 2]
```diff
@@ -1,6 +1,42 @@
...
+* Cost
+* Response format
 
 STUB
+
+### Google, Bing, and MapQuest
+[Google](https://developers.google.com/maps/documentation/geocoding/), [Bing](http://www.microsoft.com/maps/), and [MapQuest](http://www.mapquestapi.com/geocoding/) offer paid and free geocoding services. Consider subscribing to a paid service if the rate limits for the free options are too low for your application:
+
+**Rate limits**
```
Comment (malandrina) [inline book/strategies/external_services.md]: Does listing the rate limits in this section make sense? Is this information useful to readers or overwhelming?

File: book/strategies/external_services.md [hunk 3]
```diff
@@ -1,6 +1,42 @@
...
+
+### Free services
+[Nominatim](http://wiki.openstreetmap.org/wiki/Nominatim), [Yandex](http://api.yandex.com/maps/doc/jsapi/2.x/ref/reference/geocode.xml), and [ESRI](http://resources.arcgis.com/en/help/arcgis-online-geocoding-rest-api/) offer free geocoding services.
+
+**Rate limits**
+* [Nominatim](http://wiki.openstreetmap.org/wiki/Nominatim_usage_policy): 1 request/second
+* [Yandex](http://legal.yandex.com.tr/maps_api/): 25,000 requests/day
+* [ESRI](http://www.esri.com/software/arcgis/arcgisonline/services/geoservices): No limit for geosearch service
+
+### Paid services
```
Comment (malandrina) [inline book/strategies/external_services.md]: Are these worth listing?

File: book/strategies/external_services.md (hunk 1 above)
Comment (croaky) [inline book/strategies/external_services.md]: I'd leave it out for now.

File: book/strategies/external_services.md (hunk 2 above)
Comment (croaky) [inline book/strategies/external_services.md]: Maybe restrict it to a smaller list that we've actually used or recommend? Like, probably no need to include MapQuest if other options are better?

File: book/strategies/external_services.md [hunk 4]
```diff
@@ -1,6 +1,42 @@
 ## External Services
 
-* [Google](https://developers.google.com/maps/documentation/geocoding/)
-* [Yandex](http://api.yandex.com/maps/doc/jsapi/2.x/ref/reference/geocode.xml)
+### Choosing a service
```
Comment (croaky) [inline book/strategies/external_services.md]: I've been trying to consistently use title case in headers of the playbook.

Comment (malandrina) [inline book/strategies/external_services.md]: Cool, I'll follow that convention.

File: book/strategies/external_services.md (hunk 2 above)
Comment (malandrina) [inline book/strategies/external_services.md]: I agree that the list should be smaller. I'm not sure how to go about editing it, though, as I've only really interacted with the Google Geocoding API. @joshuaclayton, what do you think?

Comment (joshuaclayton) [inline book/strategies/external_services.md]: I've used Google, Yandex, and Yahoo; I think that'd be a good start. Maybe we should ping other devs to see if anyone likes other services (and for what reasons).

File: book/strategies/external_services.md (hunk 3 above)
Comment (joshuaclayton) [inline book/strategies/external_services.md]: Geocoder is good, and we should mention Google Maps for Business (even though it's expensive)

File: book/strategies/external_services.md (hunk 2 above)
Comment (malandrina) [inline book/strategies/external_services.md]: Sounds good. I asked devs in the Code Room for feedback on geocoding services they've used, but the only person to respond was Derek. Maybe we should do a more formal poll?

**Pull request merged**